scripts/generate-proxy-api-models.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+docker run --rm -v "$PWD:/usr/src/app" -e "NODE_ENV=development" -w "/usr/src/app" italia-backend/tools:1.0.0 yarn generate:proxy-api-models
```

Comment:
why run this inside a docker container?

Reply:
it is just a shortcut if the user doesn't have node installed locally